packages/vitest/src/node/core.ts

```diff
@@ -493,8 +493,8 @@ export class Vitest {
     return code.includes('import.meta.vitest')
   }
 
-  printError(err: unknown) {
-    return printError(err, this)
+  printError(err: unknown, fullStack = false) {
+    return printError(err, fullStack, this)
```

Comment:
I think we should show all stack trace for some errors (like unhandled ones), bacause it's really hard to debug otherwise.